sound/soc/sof/topology.c

```diff
@@ -1275,11 +1283,23 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	swidget->id = w->id;
 	swidget->pipeline_id = index;
 	swidget->private = NULL;
+	swidget->num_source_pins = 1;
+	swidget->num_sink_pins = 1;
+
+	ret = sof_parse_tokens(scomp, swidget, comp_pin_tokens,
+			       ARRAY_SIZE(comp_pin_tokens), priv->array,
+			       le32_to_cpu(priv->size));
+	if (ret < 0) {
+		dev_err(scomp->dev, "failed to parse component pin tokens for %s\n",
+			w->name);
+		return ret;
+	}
 
-	dev_dbg(scomp->dev, "tplg: ready widget id %d pipe %d type %d name : %s stream %s\n",
+	dev_dbg(scomp->dev,
+		"tplg: ready widget id %d pipe %d type %d name: %s stream %s pins: (%d, %d)\n",
```

Comment:
Either drop the brackets
`"tplg: ready widget id %d pipe %d type %d name: %s stream %s, pins: %d, %d\n"`
or move the pins inside:
`"tplg: ready widget id %d pipe %d type %d name: %s stream %s (pins: %d, %d)\n"`

sidenote: I always found these prints a bit confusing:
```
tplg: ready widget id 51 pipe 30 type 6 name : PGA30.0 stream none
```
this one tells that the `ready widget`'s id is 51, some words with numbers, likely pairs, then the name is `PGA30.0 stream none`
commas, colons can make a huge difference...
```
tplg: widget ready: id: 51, pipe 30, type: 6, name: PGA30.0, stream: none
```
I think I'm going to scan all prints and make a PR to sanitize them.